core/src/test/scala/kafka/server/KafkaRequestHandlerTest.scala

```diff
@@ -698,4 +698,81 @@ class KafkaRequestHandlerTest {
     // cleanup
     brokerTopicStats.close()
   }
+
+  @Test
+  def testRequestThreadMetrics(): Unit = {
+    val time = Time.SYSTEM
+    val metricsBroker = new RequestChannelMetrics(java.util.Set.of[ApiKeys])
+    val metricsController = new RequestChannelMetrics(java.util.Set.of[ApiKeys])
```

Comment:
The other tests use a `mock(classOf[RequestChannelMetrics])`. Why aren't we doing the same thing here?